### DISCUSSION TITLE
FLAME  Integration with Kubeflow Katib

### DISCUSSION THREAD

**Discussion by akramIOT** [cisco-open/flame#324] [Ideas]
Since  FLAME is  supporting  workloads to  run in a  de-coupled manner with the  data plane workload  running in edge nodes, its important to support Hyper Parameter tuning for DL in an optimum manner. In this specific context, is there a plan to support  any API level  integration with  MLOps tools like  1) Kubeflow and  2) If support for Kubeflow is there/planned  in roadmap then can we have an API from  FLAME  control plane/API server  to  tweak/parametrize  Kubelfow Katib component for  HP  tuning  to improve model efficacy ?

**Comment by myungjin** [comment]
@akramIOT Thanks for sharing your ideas. Hyperparameter tuning and NAS are indeed important topics for DL. Having said that, integration with Kubeflow is not our priority in our roadmap. Also, we need to evaluate Katib more carefully before we set up a roadmap for Katib. If we have the roadmap, we will publicize it through this thread.

**Reply by akramIOT** [reply]
> @akramIOT Thanks for sharing your ideas. Hyperparameter tuning and NAS are indeed important topics for DL. Having said that, integration with Kubeflow is not our priority in our roadmap. Also, we need to evaluate Katib more carefully before we set up a roadmap for Katib. If we have the roadmap, we will publicize it through this thread.

Thanks @myungjin
